### DISCUSSION TITLE
oqs provider configuration on windows using visual studio

### DISCUSSION THREAD

**Discussion by spellbam** [open-quantum-safe/oqs-provider#496] [Q&A]
I have configured oqs provider in openssl 3.2.2 and oqsprovider.dll is located at C:\Users\xyz\Desktop\openssl\openssl-3.2.2\ossl-modules
after follwing instructions from here https://github.com/open-quantum-safe/oqs-provider?tab=readme-ov-file

upon running this command "openssl list -providers"
I am getting
only default providers and not oqs provider

I have cloned oqs-provider with tag number as 0.6.1
and liboqs with tag number as 0.10.1

**Comment by baentsch** [comment]
> after follwing instructions from here https://github.com/open-quantum-safe/oqs-provider?tab=readme-ov-file

Did you also follow https://github.com/open-quantum-safe/oqs-provider/blob/main/USAGE.md#activation?

Be sure to checks the edits to your `openssl.cnf` file: It seems it has not been correctly changed to activate `oqsprovider`.

**Comment by baentsch** [comment]
Closing as duplicate to #497
